admin/c2cgeoportal_admin/views/layers_wms.py

```diff
@@ -65,41 +29,15 @@ class LayerWmsViews(AbstractViews):
             'ogc_server',
             renderer=lambda layer_wms: layer_wms.ogc_server.name,
             sort_column=OGCServer.name,
-            filter_column=OGCServer.name),
-        _list_field(
-            'interfaces',
-            renderer=lambda layer_wms: ', '.join([i.name or '' for i in layer_wms.interfaces]),
-            sort_column=Interface.name,
-            filter_column=Interface.name),
-        _list_field(
-            'dimensions',
-            renderer=lambda layer_wms: '; '.join(
-                ['{}: {}'.format(group[0], ', '.join([d.value for d in group[1]]))
-                 for group in groupby(layer_wms.dimensions, lambda d: d.name)])),
-        _list_field(
-            'parents_relation',
-            renderer=lambda layer_wms:', '.join([p.treegroup.name or ''
-                                                 for p in layer_wms.parents_relation])),
-        _list_field(
-            'restrictionareas',
-            renderer=lambda layer_wms: ', '.join([r.name or '' for r in layer_wms.restrictionareas])),
-        _list_field(
-            'metadatas',
-            renderer=lambda layer_wms: ', '.join(['{}: {}'.format(m.name, m.value) or ''
-                                                  for m in layer_wms.metadatas]))]
+            filter_column=OGCServer.name)
+    ] + DimensionLayerViews._extra_list_fields
     _id_field = 'id'
     _model = LayerWMS
     _base_schema = base_schema
 
     def _base_query(self):
-        return self._request.dbsession.query(LayerWMS).distinct(). \
-            join('ogc_server'). \
-            join('interfaces'). \
-            options(subqueryload('interfaces')). \
-            options(subqueryload('dimensions')). \
-            options(subqueryload('restrictionareas')). \
-            options(subqueryload('metadatas')). \
-            options(subqueryload('parents_relation').joinedload('treegroup'))
+        return super()._base_query(
```

Comment:
join on 'ogc_server' has been lost, it seems not to forbid filtering on ogc_server, but sorting !

Reply:
Done, with a test added